### PR TITLE
[smtp-server] correct type of SMTPServerSession.remoteAddress property

### PR DESCRIPTION
**Pull request by twawszczak**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
https://github.com/nodemailer/smtp-server/blob/62c78e1acecd00191015fb99c2ef5559c02d4066/lib/smtp-connection.js#L71

https://github.com/nodemailer/smtp-server/blob/62c78e1acecd00191015fb99c2ef5559c02d4066/lib/smtp-connection.js#L72

https://github.com/nodemailer/smtp-server/blob/62c78e1acecd00191015fb99c2ef5559c02d4066/lib/smtp-connection.js#L64

https://nodemailer.com/extras/smtp-server/#session-object

### CODE REVIEW

**Comment by andy-ms** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

**Review by dex4er — APPROVED** [review]
All is right

**Pull request merged**
